main.js

```diff
@@ -0,0 +1,10 @@
+let answer = "東京";
+
+while(answer == "東京"){
```

Comment:
以下のようなコードにすることで無限に繰り返すコードにできます。

```suggestion
while(true){
```